scripts/heat/instances.yaml

```diff
@@ -39,6 +39,16 @@ resources:
       flavor: { get_param: flavor }
       networks:
         - port: { get_resource: server_port }
+      user_data_format: RAW
+      user_data:
+        #!/bin/sh
+        cat >> /usr/bin/ping_forever << EOF
+        #!/bin/sh
+        trap "" 1 2 3 15
+        ping 8.8.8.8 | tee /var/log/ping_outside.out
```

Comment:
Yes, it's possible. This gets you the VM's floating IP:  `{ get_attr: [ server_floating_ip, floating_ip_address ] }`
Since this setup is split across three templates, you'll need an outputs section entry in each template to pass it all the way out to where Crowbar can get at it.